src/gpl/src/initialPlace.cpp

```diff
@@ -41,6 +43,40 @@
 #include "graphics.h"
 
 #include "utl/Logger.h"
+#include <cuda.h>
+#include <cuda_runtime.h>
+#include "cusparse.h"
+#include "cusolverDn.h"
+#include "cusolverSp.h"
+
+#define CUDA_ERROR(value) do { \
+   cudaError_t m_cudaStat = value; \
+   if(m_cudaStat != cudaSuccess){ \
```

Comment:
Style: below, you use a space between `if` and the opening parenthesis, here not. Best is to adapt it to the style that is already in the file and use that consistently.